source/_integrations/roborock.markdown

```diff
@@ -1,43 +1,20 @@
 ---
 title: Roborock
-description: Connect and control your Roborock devices using the Xiaomi Miio integration
+description: Instructions on how to integrate your Roborock vacuum with home assistant
```

Comment:
```suggestion
description: Instructions on how to integrate Roborock vacuums into Home Assistant
```
I'm not sure where/if this string is used, but this should likely be in plural form?